core/src/tvu.rs

```diff
@@ -81,7 +82,6 @@ pub struct TvuSockets {
     pub ancestor_hashes_requests: UdpSocket,
 }
 
-#[derive(Default)]
```

Comment:
This was unfortunate, but `NonZeroUsize` does not implement `Default`. Overall, this wasn't too bad and I like the strictness that using `NonZeroUsize` gives us ... we shouldn't be trying to make a thread pool with 0 threads